### PR TITLE
Cockpit ws build

### PR DESCRIPTION
**Pull request by Gundersanne**
* [x] image-refresh openshift
 * [x] image-refresh fedora-28
 * [x] fix openshift image build (PR #9975)

### CODE REVIEW

**Comment by Gundersanne** [comment]
I tested this with Dockerfile.centos7 and `cockpit-ssh`.

File: containers/ws/install-package.sh [hunk 1]
```diff
@@ -21,15 +21,16 @@ fi
 "$INSTALLER" install -y sed
 
 arch=`uname -p`
-rpm=$(ls /container/rpms/cockpit-ws-*$OSVER.*$arch.rpm /container/rpms/cockpit-bridge-*$OSVER.*$arch.rpm || true)
+rpm=$(ls /container/rpms/cockpit-ws-*$OSVER.*$arch.rpm /container/rpms/cockpit-bridge-*$OSVER.*$arch.rpm /container/rpms/cockpit-dashboard-*$OSVER.*$arch.rpm || true)
```
Comment (martinpitt) [inline containers/ws/install-package.sh]: This commit could use a rationale - i. e. on RHEL/CentOS 7, cockpit-dashboard ships ssh.

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-91pjb.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-9941-20180830-123636/

**Comment by cockpituous** [comment]
image-refresh fedora-27 done: https://github.com/cockpituous/cockpit/commits/image-refresh-fedora-27-20180830-133939

**Comment by martinpitt** [comment]
@Gundersanne I see you pushed again -- you also need to trigger the two container/ tests. See bots/image-refresh `TRIGGERS` for the list of new image → triggered tests mapping.

**Comment by martinpitt** [comment]
@Gundersanne : That's not actually that important. In fact, we often have PRs where the "main" commit is in the middle, and some auxiliary ones (like bot image rebuilds or small fixes) are before/after.

The container/bastion test failed. When I made this proposal I didn't take in mind that we use the VM-built RPMs in the container, so the container and VM host's Fedora release really need to match. Thus this is blocked on a Fedora 28 rebuild. You can actually do this and see that it works alright (it will fail fedora-atomic, but the container ones should at least work). Argh!

**Comment by martinpitt** [comment]
Force-pushed to also move the container tests to run fedora-28, as otherwise we can't bump them (see above).

**Comment by martinpitt** [comment]
#9969 landed, this should help a bit.

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-task-centos-ci-2.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-9941-20180903-154040/

**Comment by cockpituous** [comment]
image-refresh fedora-28 done: https://github.com/cockpituous/cockpit/commits/image-refresh-fedora-28-20180903-160808

**Comment by martinpitt** [comment]
I pushed the rebuilt fedora-28 image.

**Comment by martinpitt** [comment]
Hm, for container/kubernetes we would need to rebuild the openshift image.

**Comment by martinpitt** [comment]
I fixed the openshift rebuild in PR #9975. After that lands, we need to rebuild the image here (with a Fedora 28 based cockpit/base container), and rebuild fedora-28 too as it is being rebuilt in #9972 (I'm doing that to unblock #9240). Once this is unblocked, the bot label needs to be re-added.

**Comment by martinpitt** [comment]
I included the commits from #9975, so that I can already go ahead and have the images built.

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-991kn.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-9941-20180904-194444/

**Comment by cockpituous** [comment]
image-refresh fedora-28 done: https://github.com/cockpituous/cockpit/commits/image-refresh-fedora-28-20180904-204545

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-991kn.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-9941-20180904-204949/

**Comment by cockpituous** [comment]
image-refresh openshift done: https://github.com/cockpituous/cockpit/commits/image-refresh-openshift-20180904-211818

**Comment by martinpitt** [comment]
The RHEL 7 failures look like the GnuTLS bug again. The [image build log](https://fedorapeople.org/groups/cockpit/logs/image-refresh-9941-20180904-204949/log) shows that the workaround was not applied.. !?

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-pslfb.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-9941-20180905-110404/

**Comment by cockpituous** [comment]
image-refresh openshift done: https://github.com/cockpituous/cockpit/commits/image-refresh-openshift-20180905-113636

**Comment by martinpitt** [comment]
Pushed the new openshift image, this now has the workaround in place. I think that was a rebase error on the prerequisite fix.

**Comment by martinpitt** [comment]
Green enough, particularly f-28 :) The rest is unrelated to openshift, or the known kubevirt flake.

**Pull request merged**